### PR TITLE
change the cli createWallet and listAddresses from rpc service to local

### PR DESCRIPTION
**Pull request by winterweixiaodong**

### CODE REVIEW

File: dapp/cli/cli.go [hunk 1]
```diff
@@ -344,58 +350,73 @@ func getBalanceCommandHandler(ctx context.Context, client interface{}, flags cmd
 }
 
 func createWalletCommandHandler(ctx context.Context, client interface{}, flags cmdFlags) {
-	walletRequest := rpcpb.CreateWalletRequest{}
-	walletRequest.Name = "getWallet"
-	response, err := client.(rpcpb.RpcServiceClient).RpcCreateWallet(ctx, &walletRequest)
+	empty, err := logic.IsWalletEmpty()
+	//if err != nil {
```
Comment (yeeshi) [inline dapp/cli/cli.go]: code should be cleaned

File: dapp/cli/cli.go [hunk 2]
```diff
@@ -344,58 +350,73 @@ func getBalanceCommandHandler(ctx context.Context, client interface{}, flags cmd
...
-	}
-	if len(response.Address) > 0 {
-		fmt.Println("Create Wallet, the address is ", response.Address)
-	}
 	return
-
 }
 
-func listAddressesCommandHandler(ctx context.Context, client interface{}, flags cmdFlags) {
+func listAddressesCommandHandler(ctx context.Context, client1 interface{}, flags cmdFlags) {
```
Comment (yeeshi) [inline dapp/cli/cli.go]: what is the purpose to change this argument name?

File: dapp/cli/cli.go (hunk 1 above)
Comment (winterweixiaodong) [reply dapp/cli/cli.go]: done, thanks

File: dapp/cli/cli.go (hunk 2 above)
Comment (winterweixiaodong) [reply dapp/cli/cli.go]: argument name use the same name of package, this function need to use  the packet "client"

Comment (winterweixiaodong) [reply dapp/cli/cli.go]: package client, not packet

**Pull request merged**